docs/tutorials/flight_controller_telemetry_tutorial.rst

```diff
@@ -158,9 +165,15 @@ seen in the `ArduCopter code for BLHeli telemetry parsing <https://github.com/Ar
 
     ERPM to RPM Conversion
 
+Please note that you will still have to configure your flight controller to know the correct number of pole pairs present on your module. Using Mission Planner, you can set this value 

```

Comment:
Is there anyway for a customer to know the correct number of poles of the motor? Is that on the datasheet somewhere? Or should we maybe add it to the family page or something? Right now as a customer I wouldn't really know how to do this